code/modules/mob/living/silicon/pai/emote.dm

```diff
@@ -0,0 +1,163 @@
+/mob/living/silicon/pai/emote(var/act,var/m_type=1,var/message = null, var/auto)
+	if(timestopped)
+		return //under effects of time magick
+	var/param = null
+	if (findtext(act, "-", 1, null))
```

Comment:
Don't exaggerate, Regex is easy enough to learn once you piece together the basics. A match like this would be as simple as "(.*?)-(.*)". If you want to go the non-Regex route, you should still be storing the findtext result in a var instead of calling it twice.